app/modules/users/permissions/rules.py

```diff
@@ -119,26 +119,27 @@ def _can_user_perform_action(self, user):
 
         has_permission = False
         if user_is_privileged(user):
-            has_permission = True
+            # Organizations and Projects not supported for MVP, no-one can create them
+            if not self._is_module((Organization, Project)):
+                has_permission = True
         elif self._action is AccessOperation.READ:
             if user.is_admin:
                 # This is where we can control what modules admin users can and cannot read
                 # This could be Client configurable if required by taking the list passed to
                 # _is_module from a config parameter e.g. app.config.get(ADMIN_READ_MODULE_PERMISSION)
                 has_permission = self._is_module(User)
             if not has_permission and user.is_researcher:
-                has_permission = self._is_module(
-                    (Submission, Project, Encounter, Sighting, Asset)
-                )
+                has_permission = self._is_module((Submission, Encounter, Sighting, Asset))
 
         elif self._action is AccessOperation.WRITE:
-            if self._is_module((Organization, HoustonConfig)):
+            if self._is_module(HoustonConfig):
                 has_permission = user.is_admin
             elif self._is_module((Submission, User)):
                 # Any users can submit and write (create) a user
                 has_permission = True
-            elif self._is_module(Project):
-                has_permission = user.is_researcher
+            # Project disabled for MVP
```

Comment:
This temporary disabling is mostly implemented by commenting out existing permissions code, something I normally tell people not to do.
Is this "the right way to do it", "always the wrong way to do it" or "acceptable.... just, in this instance"?

Reply:
I don't think there is a more-right way to do it, and this is fine. We shouldn't overthink disabling this functionality... we will hopefully be working on adapting and re-enabling it soon after MVP. 

There shouldn't be too much much concern over leaving small parts active. I think there is little harm they can do, especially since they were never enabled in a production environment so the chances of the code being run are slim.